packages/design-system/src/components/react-aria/index.js

```diff
@@ -1,13 +1,56 @@
 import $73SJx$react, { useState, useRef, useEffect, useCallback, useMemo, useContext } from 'react';
 import $12uGp$clsx from 'clsx';
-import { _ as _$1 } from '@swc/helpers/_/_class_private_field_get';
-import { _ as _$2 } from '@swc/helpers/_/_class_private_field_init';
-import { _ } from '@swc/helpers/_/_class_private_field_set';
```

Comment:
If this is the only usage, should the dependency be removed rather than commented out? Or is the intention to reinstate them at some point?

Reply:
Ah, sorry, I think I'm turned around from looking through various package.json's yesterday and assumed you guys had `@swc/helpers` as a direct dependency rather than a child dependency of `react-aria`.
Although, if the code does directly leverage the package _should_ it be a direct dependency? Perhaps the fact that it was being consumed as a dependency of a dependency was a cause for npm's difficulty to resolve the correct package version for us 🤷‍♂️

Reply:
That's possible. I copied the source of `react-aria` into here so I could make a fix for Preact that they couldn't accept. I made that decision because of the short timeline we're facing for the release, but I want to keep the direct dependencies lean.